packages/components/CHANGELOG.md

```diff
@@ -30,6 +30,7 @@
 -   `AutoComplete`: Revert recent `exhaustive-deps` refactor ([#41820](https://github.com/WordPress/gutenberg/pull/41820)).
 -   `Spacer`: Convert knobs to controls in Storybook ([#41851](https://github.com/WordPress/gutenberg/pull/41851)).
 -   `Heading`: Complete TypeScript migration ([#41921](https://github.com/WordPress/gutenberg/pull/41921)).
+-   `Divider`: Complete TypeScript migration ([#41991](https://github.com/WordPress/gutenberg/pull/41991)).
```

Comment:
We may need to rebase and add this CHANGELOG entry to the new `Unreleased` section, since version `19.14.0` was released yesterday

Reply:
Will update 👍